File: Android/app/src/main/java/com/didichuxing/doraemondemo/LeakActivity.java
package com.didichuxing.doraemondemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

/**
 * 模拟内存泄漏的activity
 */
public class LeakActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_leak);
        App.leakActivity = this;
    }

}

File: Android/app/src/main/java/com/didichuxing/doraemondemo/LeakActivity.kt
package com.didichuxing.doraemondemo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

/**
 * 模拟内存泄漏的activity
 */
class LeakActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_leak)
        App.leakActivity = this
    }
}